src/main/java/com/CezaryZal/controller/UserHcController.java

```diff
@@ -0,0 +1,24 @@
+package com.CezaryZal.controller;
+
+import com.CezaryZal.entity.health.calendar.UserAuthentication;
+import org.springframework.http.ResponseEntity;
+
+import javax.naming.NameNotFoundException;
+import java.util.List;
+
+public interface UserHcController {
+
+    ResponseEntity<UserAuthentication> findById(Long id) throws NameNotFoundException;
+
+    ResponseEntity<UserAuthentication> findByLoginName(String loginName) throws NameNotFoundException;
+
+    boolean isExistsUserAuthenticationByLoginName(String loginName);
+
+    List<UserAuthentication> findAll();
+
+    UserAuthentication addNewUser(UserAuthentication userAuthentication);
+
```

Comment:
UserAuthentication to encja, raczej nie powinna pojawić się ani w request ani w response

Reply:
W requescie, racja ale w response czemu nie?
Tutaj są endpointy dla aministratora żeby można było bezpośrednio komunikować się z repozytorium.

Reply:
Bo jakakolwiek zmiana obiektu encji wymusza zmiane API i w drugą strone, zmiana API wymusza zmianę struktury bazy danych

Reply:
https://devszczepaniak.pl/wstep-do-rest-api/ pkt.4